harness/determined/cli/util.py

```diff
@@ -26,30 +27,26 @@
     ),
 }
 
-pagination_args_fetchone = [
-    Arg(
-        "--offset",
-        type=int,
-        default=0,
-        help="Offset the returned set.",
-    ),
-    Arg(
-        "--limit",
-        type=int,
-        default=50,
-        help="Limit the returned set.",
-    ),
-    Arg(
-        "--reverse",
-        default=False,
-        action="store_true",
-        help="Reverse the requested order of results.",
-    ),
-]
+
+class PageOpts(str, Enum):
```

Comment:
these options are pretty limiting, but i guess that is fine.

Reply:
For unlimited custom pagination I'd rather just use the bindings directly.